types/events.go

```diff
@@ -55,10 +55,12 @@ type Throttling struct {
 
 type CpuUsage struct {
 	// Units: nanoseconds.
-	Total  uint64   `json:"total,omitempty"`
-	Percpu []uint64 `json:"percpu,omitempty"`
-	Kernel uint64   `json:"kernel"`
-	User   uint64   `json:"user"`
+	Total        uint64   `json:"total,omitempty"`
+	Percpu       []uint64 `json:"percpu,omitempty"`
+	PercpuKernel []uint64 `json:"percpuKernel,omitempty"`
+	PercpuUser   []uint64 `json:"percpuUser,omitempty"`
```

Comment:
These JSON fields should use underscores (`percpu_kernel` and `percpu_user`). I know that some others like `kernelTCP` don't but in theory that's the style for event JSON.